build-system/amp4test.js

```diff
@@ -186,7 +186,11 @@ app.use('/request-bank/:bid/teardown/', (req, res) => {
  */
 app.get('/a4a/:bid', (req, res) => {
   const {bid} = req.params;
-  const body = `
+  const preSet = `
```

Comment:
to not introduce custom js in the ad, you can set the cookie via HTTP response header

see https://expressjs.com/en/api.html#res.cookie

Reply:
Sounds like a better solution. Let me try that

Reply:
done